src/chttpd/src/chttpd_util.erl

```diff
@@ -95,3 +96,20 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+get_dbs_info_shards(DbName) ->
+    Timeout = fabric_util:request_timeout(),
+    IsolatedFun = fun() -> fabric:get_db_info(DbName) end,
+    try fabric_util:isolate(IsolatedFun, Timeout) of
+        {ok, DbInfo} ->
+            {info, {DbInfo}};
```

Comment:
Here it looks like we're trying to return a ready-made json response, but we also return an `{error, _}` tuple which can't be serialized to json. I think we should pick one or the other. Perhaps stick to the common `{ok, _} | {error, _}` shape so we can re-use this function elsewhere.